cluster-autoscaler/core/autoscaler.go

```diff
@@ -20,53 +20,24 @@ import (
 	"strings"
 	"time"
 
-	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
 	cloudBuilder "k8s.io/autoscaler/cluster-autoscaler/cloudprovider/builder"
-	"k8s.io/autoscaler/cluster-autoscaler/config"
 	"k8s.io/autoscaler/cluster-autoscaler/context"
+	"k8s.io/autoscaler/cluster-autoscaler/core/options"
```

Comment:
let's use `coreoptions` alias everywhere we import this to be consistent

Reply:
updated these references